client/src/components/Preloader/Preloader.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import classes from './preloader.module.css';
+import sun from '../../../public/images/sun.png';
+import moon from '../../../public/images/moon.png';
+import { string } from 'prop-types'
+
+function Preloader({ centralObj, rotatingObj }) {
+  return (
+    <div className={classes.container}>
+      <img src={centralObj} className={classes.centralObj} alt='cental obj'/>
+      <img src={rotatingObj} className={classes.rotatingObj} alt ='rotating obj'/>
+    </div>
+  );
+}
+
+Preloader.defaultProps = {
+    centralObj: moon,
```

Comment:
it's not necessarily a good practice to use position related names for your objects